src/App.tsx

```diff
@@ -177,6 +177,7 @@ function AppContent() {
   const [showNavBar, setShowNavBar] = useState(true);
   const trackingV2 = isEnabled('TrackingV2');
   const dispatch = useAppDispatch();
+  const { user } = useUser();
```

Comment:
i believe we use `useLocalization` and pick `activeLocale` from that

Reply:
Should I change it on every component? What is the difference with the user locale?

Reply:
I believe most components already uses useLocalization. If you just change it here in your PR that should be ok.
Objective was that localization information is provided by the useLocalization provider. This allows us to not depend on internal semantics such as user.locale.